src/poliastro/tests/test_frames.py

```diff
@@ -124,8 +124,8 @@ def test_round_trip_from_GeocentricSolarEcliptic_gives_same_results():
 def test_GeocentricSolarEcliptic_against_data():
     gcrs = GCRS(ra="02h31m49.09s", dec="+89d15m50.8s", distance=200 * u.km)
     gse = gcrs.transform_to(GeocentricSolarEcliptic(obstime=Time("J2000")))
-    lon = 233.11663895663975
-    lat = 48.64652559835358
+    lon = 233.1169136260286
```

Comment:
Do we know _why_ the numbers are different here?

Reply:
They were failing with my previous local caché but with the updated one this is not showing as a failing test anymore.

Reply:
See my comment below.